CONTRIBUTING.md

```diff
@@ -91,8 +91,30 @@ General requirements for submitting rules: don't add rules to the beginning of t
 
 ### AdGuard Annoyances filter
 
-* Purpose: this filter blocks various kinds of annoyances from sites.
-* [Annoyances folder](https://github.com/AdguardTeam/AdguardFilters/tree/master/AnnoyancesFilter/sections)
+* Purpose: Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups.
+* [Annoyances folder](https://github.com/AdguardTeam/AdguardFilters/tree/master/AnnoyancesFilter)
+
+Contains the following AdGuard filters: Cookie Notices, Popups, Mobile App Banners, Other Annoyances and Widgets:
+
+- ### Cookie Notices
```

Comment:
@zloyden Do I need to indent the tabs for the sub-items?

Reply:
I guess we can do it like
```
Contains the following AdGuard filters:

- ### AdGuard Cookie Notices
```

Reply:
I'd prefer to do it.
How it looks with tabs:
<details>
<summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/8361299/178263116-a3624fe0-a5ec-4e00-a270-4639b4fdfb40.png)

</details>

Reply:
I think, that it is not necessary to write the full names, as these are the components of the annoyance filter

Reply:
Also, text looks more readable with tabs. If we write "AdGuard" at the beginning of each section, we lose the visual recognition that it is a sub-item.

Reply:
As I can see, there are two options.
One of them - do tabs like [here](https://github.com/AdguardTeam/AdguardFilters/pull/124043#discussion_r917867865) without adding full names because we have it in description.